README.md

```diff
@@ -1,5 +1,5 @@
 # OSM Map Widget for Elementor
-[A free Elementor Map Widget](https://wordpress.org/plugins/osm-map-elementor/) that utilizes [Open Street Map](https://www.openstreetmap.org/). Comes with features like adding multiple markers, and choosing from a library of custom tiles to change the look and feel. Requires [Elementor Page Builder](https://wordpress.org/plugins/elementor/) Plugin Version: 3.2+
+[A free Elementor Map Widget](https://wordpress.org/plugins/osm-map-elementor/) that utilizes [Open Street Map](https://www.openstreetmap.org/). Comes with features like adding multiple markers, and choosing from a library of custom tiles to change the look and feel. Requires [Elementor Page Builder](https://wordpress.org/plugins/elementor/) Plugin Version: 3.5+
```

Comment:
@QuoterMusic - What are your thoughts of updating the **Open Street Map** link here to **[OpenStreetMap](https://www.openstreetmap.org/copyright)** _(Point to OSM  copyright & license page)_

Reply:
Would be great to also to add the same link under the **[License](https://github.com/QuoterMusic/osm-map-elementor/tree/main#licence)** section of the README

Reply:
I don't think this is necessary if I would rather suggest the [About page](https://www.openstreetmap.org/about) where you can find all the info about OSM

Reply:
I actually do not have access to the OSM [About page](https://www.openstreetmap.org/about). I can only make edits to the plugin's Wordpress landing page 😃